main/src/ca/uwaterloo/flix/tools/pkg/ManifestParser.scala

```diff
@@ -218,18 +217,33 @@ object ManifestParser {
       try {
         toSemVer(depVer.asInstanceOf[String], p) match {
           case Ok(version) => if (flixDep) {
-            if (prodDep) {
-              depsSet.add(Dependency.FlixDependency(depName, version, DependencyKind.Production))
+            val depNameSplit1 = depName.split(':')
```

Comment:
I am wondering if it would not be better to split into a list and then simply do

```
... match {
case host :: user :: project => ...
```